component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/client/MoesifClient.java

```diff
@@ -82,15 +82,28 @@ private void doRetry(String orgId, MetricEventBuilder builder) {
     public void publish(MetricEventBuilder builder) throws MetricReportingException {
         Map<String, Object> event = builder.build();
         ConcurrentHashMap<String, String> orgIDMoesifKeyMap = keyRetriever.getMoesifKeyMap();
+        ConcurrentHashMap<String, String> orgIdEnvMap = keyRetriever.getEnvMap();
+        LinkedHashMap properties = (LinkedHashMap) event.get("properties");
```

Comment:
Make this a constant.

Reply:
Resolved with: 9a0b30dc39ce6c0986d843daf6803adebbe92e56